### PR TITLE
Development

### PR DESCRIPTION
**Pull request by farouqdaif1**
- Choose the topic for your website and the API.
- Choose images, text, icons, fonts.
- Set up the repository and tools.
- Create the basic shared code on the home page.
- Start individual interfaces.
- Give code reviews to your teammates.
- Work on individual interfaces.
- Give code reviews to your teammates.
- Complete the work on individual interfaces.
- Record a video for your project.
- Create a good README and PR description.
- Ask for the external review - using your Student Dashboard.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,85 @@
...
+In this project we will prepare a Kanban board with a GitHub project, with my Learning partner. It's javascript capstone project with external API..
+
+![image] ![screenshot](photo.jpg)
+
+## Video link
+
+[loom Link] (https://loom.com/share/cebf106ada4f4ac789399a6eb27dd385)
+
+## Live Demo
+
```
Comment (SajjadAhmad14) [inline README.md]: Please add an active live demo link or remove this option from here.

File: README.md [hunk 2]
```diff
@@ -1 +1,85 @@
...
+- Follow the instructions from the [getting started](https://webpack.js.org/guides/getting-started/#basic-setup) guide to set up the basics. Implement all the steps from _Basic Setup_ to _NPM Scripts_.
+- The next step is to add html file :
+  Follow the instructions from the [setting up HtmlWebpackPlugin](https://webpack.js.org/guides/output-management/#setting-up-htmlwebpackplugin) guide
+  Then
+
+```
+npm run build
+```
+
```
Comment (SajjadAhmad14) [inline README.md:35]: Please add `npm install` and `npm test` commands.

File: README.md [hunk 3]
```diff
@@ -1 +1,85 @@
-# CapstoneJS
\ No newline at end of file
+# CapstoneJS
+
+In this project we will prepare a Kanban board with a GitHub project, with my Learning partner. It's javascript capstone project with external API..
+
+![image] ![screenshot](photo.jpg)
+
+## Video link
```
Comment (SajjadAhmad14) [inline README.md:7]: All the members should appear in the video and talk about the project.
The video should contain a highlight about a piece of the code that you find interesting or are proud of making.

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        <section class=home-page>
+            <ul class="meals">
+            </ul>
+        </section>
+    </main>
+    <footer>
+        <h3>
+            <span>Created By</span>
+            <a href="https://github.com/farouqdaif1">Farouq Daif</a>
+            &amp;
```
Comment (SajjadAhmad14) [inline src/index.html:40]: Please follow the mock-up for the Home page footer. It should be on the left side of the page.

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&family=Mr+De+Haviland&family=Roboto&display=swap" rel="stylesheet">
+    <title>Chrimpo</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="logo"> <a href="index.html">chrimpo</a></div>           
+                <ul class="nav-list">
+                    <li><a id="seafood-meals" href="index.html"> Seafood meals</a></li>
```
Comment (SajjadAhmad14) [inline src/index.html]: The home page should have a counter which shows the number of meals shown on the page. Please refer to the wireframe to implement this feature.

File: src/likes.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Jxa2o7XJAhJoB2RZ7ghK/likes';
+// const id = Jxa2o7XJAhJoB2RZ7ghK;
+const addLikes = (id) => fetch(baseUrl, {
+  method: 'POST',
```
Comment (SajjadAhmad14) [inline src/likes.js]: Please remove commented code from `likes.js` and from all other files wherever present as it can mess up the whole code base.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Aku and Farouq, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
UI is attractive and fascinating. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (aakbarkhan) [reply README.md]: Thank you.

File: README.md (hunk 3 above)
Comment (aakbarkhan) [reply README.md:7]: Changes made.

File: src/index.html (hunk 4 above)
Comment (aakbarkhan) [reply src/index.html:40]: Changes Done. Thank you.

File: src/likes.js (hunk 6 above)
Comment (aakbarkhan) [reply src/likes.js]: Removed.

File: src/index.html (hunk 5 above)
Comment (aakbarkhan) [reply src/index.html]: Fixed, thank you.

**Comment by omar-labana** [comment]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (farouqdaif1) [reply README.md]: Thank you

File: README.md (hunk 2 above)
Comment (farouqdaif1) [reply README.md:35]: Thank you

File: src/index.html (hunk 4 above)
Comment (farouqdaif1) [reply src/index.html:40]: Thank you

File: src/index.html (hunk 5 above)
Comment (farouqdaif1) [reply src/index.html]: Thank you

**Pull request merged**